src/data-layer/TwitchUserData.js

```diff
@@ -29,23 +29,24 @@ class TwitchUserData extends ServerCacheManager {
     return this.get(
       _id,
       TwitchUserData.getDefaultTwitchUserData(TwitchUserAPIData)
-    ).then((TwitchUserDBData) => {
```

Comment:
As far as I can tell, this is just a slight formatting change. In the future, try to keep formatting changes in separate pull requests... otherwise, pull requests can become large and a burden to sift through for other developers.